press/legacy_publishing/litezip.py

```diff
@@ -67,8 +61,8 @@ def publish_litezip(struct, submission, db_conn):
         except Unchanged:
             pass  # only publish content that has changed.
 
-    any_changes = id_map != {}
-    if any_changes:
+    collxml_changed = bool(id_map)
```

Comment:
Nice! bool(id_map) is much cleaner than what I had. But, an empty id_map actually means that no modules changed, so I'd suggest renaming collxml_changed to modules_changed.

Reply:
Yes, I think that makes sense.  I'll change it. :+1: